CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0-alpha4] - (unreleased)
+
+### Fixed
+
+- Using a TypeScript build now, so everything we ship is vanilla JS.
```

Comment:
I'd add a comment to the effect of: this enables support for older versions of React Native (and perhaps add that we've verified on 0.54 and newer or something)